lib/eventFactory.js

```diff
@@ -232,6 +233,15 @@ function createBuilds(config) {
         return null;
     }
 
+    // enable pipeline parameters feature
+    if (pipeline.parameters && pipelineConfig.meta && pipelineConfig.meta.parameters) {
+        const allowedPipelineParameters = _.keys(pipeline.parameters);
+        const currentPipelineParamters = _.assign({}, pipeline.parameters,
+            pipelineConfig.meta.parameters);
+
+        pipelineConfig.meta.paramters = _.pick(currentPipelineParamters, allowedPipelineParameters);
```

Comment:
mispelled `parameters` here. 
can you add test for this?

Reply:
Corrected it.